lms/envs/common.py

```diff
@@ -4970,3 +4970,25 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     "ruby", "s", "samp", "section", "small", "source", "span", "strong", "style", "sub", "sup", "table", "tbody", "td",
     "tfoot", "th", "thead", "time", "tr", "track", "u", "ul", "var", "video", "wbr",
 ]
+
+################ Settings for the Discussion Service #########
+# Provide a list of reason codes for moderators editing posts and
+# comments, as a mapping from the internal reason code representation,
+# and an internationalizable label to be shown to moderators in the form
+# UI.
+DISCUSSION_MODERATION_EDIT_REASON_CODES = {
+    "grammar-spelling": _("Has grammar / spelling issues"),
+    "needs-clarity": _("Content needs clarity"),
+    "academic-integrity": _("Has academic integrity concern"),
+    "inappropriate-language": _("Has inappropriate language"),
+    "contains-pii": _("Contains personally identifiable information"),
+}
+# Provide a list of reason codes for moderators to close posts, in the
+# form of tuples containing the internal reason code representation, and
```

Comment:
@xitij2000 This comment seems outdated. This is no longer a tuple. It should say `as a mapping` similar to the comment above EDIT REASON CODES